workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6

```diff
@@ -3086,25 +3148,37 @@
           </Row>
           <Row>
             <Name>TSPC_L2CAP_2_46</Name>
-            <Description>Support LE Credit Based Flow Control Mode (C.19)</Description>
+            <Description>LE Credit Based Flow Control Mode (C.19)</Description>
             <Value>TRUE</Value>
             <Mandatory>FALSE</Mandatory>
           </Row>
           <Row>
             <Name>TSPC_L2CAP_2_47</Name>
-            <Description>Support for LE Data Channel (C.20)</Description>
+            <Description>LE Data Channel (C.20)</Description>
             <Value>TRUE</Value>
             <Mandatory>FALSE</Mandatory>
           </Row>
           <Row>
             <Name>TSPC_L2CAP_2_48</Name>
-            <Description>Support Enhanced Credit Based Flow Control Mode (C.23)</Description>
+            <Description>Enhanced Credit Based Flow Control Mode (C.23)</Description>
             <Value>TRUE</Value>
             <Mandatory>FALSE</Mandatory>
           </Row>
+          <Row>
+            <Name>TSPC_L2CAP_2_48a</Name>
+            <Description>Enhanced Credit Based Flow Control Mode - BR/EDR (C.26)</Description>
+            <Value>FALSE</Value>
+            <Mandatory>FALSE</Mandatory>
+          </Row>
+          <Row>
+            <Name>TSPC_L2CAP_2_48b</Name>
+            <Description>Enhanced Credit Based Flow Control Mode - LE (C.27)</Description>

```

Comment:
Again this should be verified through Launch studio because we do support ECFC if I recall correctly.